components/customNodes.tsx

```diff
@@ -125,22 +133,29 @@ export const Consumer = ({ data }: any) => {
         data-popover-trigger="hover"
         type="button"
         onClick={handleModalOpen}
-        className="w-[205px] h-[64px] bg-white rounded-md shadow-lg border-1 border-purple-200 flex items-center justify-between px-1"
+        className="w-[250px] h-[64px] bg-white rounded-lg shadow-lg border-1 border-purple-200 flex items-center justify-between px-1"
       >
         <IconGripVertical
           class="w-6 h-6 text-purple-100 cursor-grab"
           id="dragHandle"
         />
-        <img src={"/images/placeholder-icon.png"} className="w-[30px]" />
-        <a href={`/${data.label.toLowerCase()}`}>
-          <div className={"flex flex-col"}>
-            <h2 className={"text-[16px]"}>
-              Item Name
+
+        <ConsumerIcon className={"mx-2"} />
+        <a
+          href={`/${encodeURIComponent(data.pipeline.audience.serviceName)}/${
```

Comment:
This link need to be updated to reflect the new `/service/[service]/component/[component]..` structure.